composer.json

```diff
@@ -10,20 +10,22 @@
     ],
     "require": {
         "jms/metadata": "~1.5",
-        "symfony/property-access": "~2.4",
+        "symfony/property-access": "~2.6||~3.0",
```

Comment:
s/death/dead :) We are < 1.0 so technically we *could*, but it would break peoples installations, f.e. https://packagist.org/packages/esperanto/esperanto-cms-project is tagged to `0.2` but there may be other people not using packagist that are using `~0.2` or similar.